packages/mdc-tab-bar/index.js

```diff
@@ -0,0 +1,147 @@
+/**
+  * @license
+  * Copyright 2018 Google Inc. All Rights Reserved.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License")
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *      http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+import MDCComponent from '@material/base/component';
+
+import {MDCTab, MDCTabFoundation} from '@material/tab/index';
+import {MDCTabScroller} from '@material/tab-scroller/index';
+
+import MDCTabBarAdapter from './adapter';
+import MDCTabBarFoundation from './foundation';
+
+/**
+ * @extends {MDCComponent<!MDCTabBarFoundation>}
+ * @final
+ */
+class MDCTabBar extends MDCComponent {
+  /**
+   * @param {...?} args
+   */
+  constructor(...args) {
+    super(...args);
+
+    /** @type {!Array<!MDCTab>} */
+    this.tabList;
```

Comment:
Is there a reason that `tabList` and `tabScroller` are implicitly public? And should the others use `@private`?